db/migrate/20200212161248_add_position_to_course_subject.rb

```diff
@@ -0,0 +1,5 @@
+class AddPositionToCourseSubject < ActiveRecord::Migration[6.0]
+  def change
+    add_column :course_subject, :position, :integer
```

Comment:
Do we need a default of `0`

Reply:
Don't think so. The subjects that are in there now don't currently have any position assigned to them, so I think it's ok for them to be `null`. Once we backfill values, maybe then it'll make sense to default to 0.